components/director/internal/tenantfetcher/service_test.go

```diff
@@ -61,39 +62,103 @@ func TestService_SyncTenants(t *testing.T) {
 			},
 			APIClientFn: func() *automock.EventAPIClient {
 				client := &automock.EventAPIClient{}
-				client.On("FetchTenantEventsPage", tenantfetcher.CreatedEventsType, 1).Return(fixTenantEventsResponse(tenantEvents, 3, 1), nil).Once()
-				client.On("FetchTenantEventsPage", tenantfetcher.UpdatedEventsType, 1).Return(fixTenantEventsResponse(tenantEvents, 3, 1), nil).Once()
-				client.On("FetchTenantEventsPage", tenantfetcher.DeletedEventsType, 1).Return(fixTenantEventsResponse(tenantEvents, 3, 1), nil).Once()
+				client.On("FetchTenantEventsPage", tenantfetcher.CreatedEventsType, tenantfetcher.QueryParams{
```

Comment:
This `tenantfetcher.QueryParams` variable is created in-place identically throughout the test file - maybe extract it?